src/main/java/com/iota/iri/storage/rocksDB/RocksDBPersistenceProvider.java

```diff
@@ -334,10 +334,18 @@ public void deleteBatch(Collection<Pair<Indexable, ? extends Class<? extends Per
                     Indexable indexable = entry.low;
                     byte[] keyBytes = indexable.bytes();
                     ColumnFamilyHandle handle = classTreeMap.get(entry.hi);
-                    writeBatch.remove(handle, keyBytes);
+                    try {
+                        writeBatch.delete(handle, keyBytes);
+                    } catch (RocksDBException e) {
+                        log.error("Could not delete handle: " + handle.getID());
+                    }
```

Comment:
I believe that which ever service calls delete batch should catch the error and decide how to handle it.

Reply:
I wrapped it in a RuntimeException.